docs/manage-metadata-using-gravitino.md

```diff
@@ -733,6 +733,159 @@ In addition to the basic settings, Gravitino supports the following features:
+| hash            | Bucket table using hash. The data will be distributed into buckets based on the hash value of the key.               | Any          | int         | `hash`  | `Strategy.HASH`  |
+| range           | Bucket table using range. The data will be divided into buckets based on a specified range or interval of values.    | Any          | Source type | `range` | `Strategy.RANGE` |
+| even            | Bucket table using even. The data will be evenly distributed into buckets, ensuring an equal distribution of data.   | Any          | Source type | `even`  | `Strategy.EVEN` |
+
+- Number. It defines how many buckets you use to bucket the table.
+- Function arguments. It defines which field or function should be used to bucket the table. Gravitino supports the following three kinds of arguments, for more, you can refer to Java class `FunctionArg` and `DistributionDTO` to use more complex function arguments.
+
+| Expression type | Json example                                                      | Java example                                                                                            | Equivalent SQL semantics | Description                    | 
+|-----------------|-------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|--------------------------|--------------------------------|
+| field           | `{"type":"field","fieldName":["score"]}`                          | `FieldReferenceDTO.of("score")`                                                                         | `score`                  | field reference value `score`  |
+| function        | `{"type":"function","functionName":"hour","fieldName":["score"]}` | `new FuncExpressionDTO.Builder()<br/>.withFunctionName("hour")<br/>.withFunctionArgs("score").build()`  | `hour(score)`            | function value `hour(score)`   |
+| constant        | `{"type":"constant","value":10, "dataType": "integer"}`           | `new LiteralDTO.Builder()<br/>.withValue("10")<br/>.withDataType(Types.IntegerType.get())<br/>.build()` | `10`                     | Integer constant `10`          |
+
+
+<Tabs>
+<TabItem value="Json" label="Json">
+
+```json
+{
+  "strategy": "hash",
+  "number": 4,
+  "funcArgs": [
+    {
+      "type": "field",
+      "fieldName": ["score"]
+    }
+  ]
+}
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+ new DistributionDTO.Builder()
+    .withStrategy(Strategy.HASH)
+    .withNumber(4)
+    .withArgs(FieldReferenceDTO.of("score"))
+    .build()
+```
+
+</TabItem>
+</Tabs>
+
+
+#### Sorted order table
+
+To define a sorted order table, you should use the following three components to construct a valid sorted order table.
+
+- Direction.  It defines in which direction we sort the table.
+
+| Direction  | Json   | Java                       | Description                               |
```

Comment:
`| Direction| Description | Json | Java| `?

Reply:
OK.

I hesitate to modify it because `Json` and `Java` are NOT an examples and their literal meaning are very clear, we don't need to move `Description` column in the second.

Reply:
Json and Java is a specified language. Description is a more common information for users.